### PR TITLE
cov: add tests in {events,deterministic}.py

### PR DESCRIPTION
**Pull request by ydcjeff**
Part of #1790 

Description: Add tests in events.py

Check list:

- [x] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [ ] Documentation is updated (if required)

### CODE REVIEW

File: tests/ignite/engine/test_event_handlers.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 from pytest import raises
 
 from ignite.engine import Engine, Events, State
-from ignite.engine.events import EventsList
+from ignite.engine.events import EventsList, RemovableEventHandle
```
Comment (ydcjeff) [inline tests/ignite/engine/test_event_handlers.py:9]:
```suggestion
from ignite.engine.events import EventsList
```

File: tests/ignite/engine/test_deterministic.py [hunk 2]
```diff
@@ -21,6 +21,11 @@
 from tests.ignite.engine import BatchChecker, setup_sampler
 
 
+def test_dengine_setup_seed_div_by_zero():
+    with pytest.raises(ZeroDivisionError, match=r"integer division or modulo by zero"):
```
Comment (trsvchn) [inline tests/ignite/engine/test_deterministic.py:25]: Nice catch!
@vfdev-5 should we handle this `iter_counter=0` exception with our custom exception message?

Comment (ydcjeff) [reply tests/ignite/engine/test_deterministic.py:25]: But `_setup_seed` is called internally with arguments which does not occur ZeroDivError

Comment (trsvchn) [reply tests/ignite/engine/test_deterministic.py:25]: oh, I wasn't clear, I meant catch it before `Zero..` with custom `ValueError`

```python
if iter_counter == 0:
    raise ValueError
```

**Review by trsvchn — APPROVED** [review]
LGTM! Thanks @ydcjeff !

**Review by sdesrozis — APPROVED** [review]
@ydcjeff Thank you !

**Pull request merged**